.github/codecov.yml

```diff
@@ -24,3 +24,4 @@ comment:
 
 ignore:
   - "tests/*/**"		# Don't need Tests to cover themselves
+  - "contrib/zstd/**/*"		# Dependencies don't need coverage testing
```

Comment:
My "tab typo" is annoying me... mind getting rid of it when you're at it anyway?
Sorry!

Reply:
fixed ;)